### ISSUE TITLE
Links should have descriptive aria-label tags

### ISSUE THREAD

**Issue by mattcarlotta** [mattcarlotta/portfolio-astro-vercel#5]
The default for a `Link` component aria-label attribute is "Navigate to {title} page"
- Either remove the redundant text or include accessible headers that describe the links
